Add instructions configuring the default password for GreenLakeAdmin (#8)
HewlettPackard/hpegl-metal-os-windows-iso: README.md

  * [Building the Bare Metal Windows image and service](#building-the-bare-metal-windows-image-and-service)
* [Customizing the Windows image](#customizing-the-windows-image)
  * [Modifying the way the image is built](#modifying-the-way-the-image-is-built)

>   * [Default GreenlakeAdmin account password](#default-greenlakeadmin-password)

Reviewer: ```suggestion
  * [Default GreenLakeAdmin account password](#default-greenlakeadmin-password)
```

Feel free to modify these files to suit your specific needs.
General changes that you want to contribute back via a pull request are much appreciated.

> ## Default GreenlakeAdmin password

Reviewer: "Lake" is capitalized in the user data 
```suggestion
## Default GreenLakeAdmin password
```

Author: That was actually a typo. In the cloudbase-init config when we tell it what account to rename to it is "GreenlakeAdmin" so that's what the user is renamed to. I have updated the value in the user-data file to reflect what it really is and left the Readme alone.

Reviewer: I assume it is not case sensitive? I was using GreenLakeAdmin and it worked (at least the last time when I got it to meet the complexity criteria).

> These scripts support one of two ways of securing the default GreenlakeAdmin account. This is the built-in Administrator account that has been renamed.

Reviewer: ```suggestion
These scripts support one of two ways of securing the default GreenLakeAdmin account. This is the built-in Administrator account that has been renamed.
```

> 1. Statically defined password in the user-data file.

Author: ```suggestion
1. Statically defined password in the [user-data](glm-cloudbaseinit-setup.ps1.template.dos#L17-L35) file.
```

> 1. Statically defined password in the user-data file.

Reviewer 2: Does "User defined password" sound better than "Statistically defined"?

Author: Doesn't matter to me. What ever would be easier to understand by the consumer of the documentation.

> 1. Statically defined password in the [user-data](glm-user-data.template.dos#L9) file.

Author: ```suggestion
1. User defined password in the [user-data](glm-user-data.template.dos#L9) file.
```

> 1. Statically defined password in the [user-data](glm-cloudbaseinit-setup.ps1.template.dos#L17-L35) file.

Author: ```suggestion
1. Statically defined password in the [user-data](glm-user-data.template.dos#L9) file.
```

>    * This will enable a known password to be used for the GreenlakeAdmin account

Reviewer: ```suggestion
   * This will enable a known password to be used for the GreenLakeAdmin account
```

>    * This password should be set in clear text in the user-data file with the passwd property in the users section

Reviewer: ```suggestion
   * This password should be set in clear text in the user-data file with the "passwd" property in the users section
```

   * This password must meet password complexity requirements of the Windows OS (default of minimum 10 characters with a mix of upper-case, lower-case, number, and symbol)

>    * In addition, the CloudBase-Init configuration file must load UserDataPlugin at some point after CreateUserPlugin but before LocalScriptPlugin

Author: ```suggestion
   * In addition, the CloudBase-Init configuration file must load [UserDataPlugin](glm-user-data.template.dos#L30) at some point after CreateUserPlugin but before LocalScriptPlugin
```

>    * After the initial boot if Windows and giving CloudBase-Init time to run, it will set this password and you will be able to use it to log in via RDP or Serial Console SAC

Reviewer 2: "After the initial boot if Windows and giving CloudBase-Init time to run" - can you please double check this sentence?

>    * After the initial boot if Windows and giving CloudBase-Init time to run, it will set this password and you will be able to use it to log in via RDP or Serial Console SAC

Author: ```suggestion
   * After the initial boot of Windows, and giving CloudBase-Init time to run, it will set this password and you will be able to use it to log in via RDP or Serial Console SAC
```

   * The CloudBase-Init LocalScripts will then set the CloudBase-Init service to Disabled so you can change that password post-deployment without CloudBase-Init setting it back to the defined value after reboot
1. Randomly generated password

>    * If the password is not specified in the user-data file (or the password specified does not meet complexity requirements), then CloudBase-Init will set a random password for the GreenlakeAdmin user

Reviewer: ```suggestion
   * If the password is not specified in the user-data file (or the password specified does not meet complexity requirements), then CloudBase-Init will set a random password for the GreenLakeAdmin user
```

   * With this setting, the UserDataPlugin is not needed

>    * It is also not necessary to disable the CloudBase-Init service after inital boot, however there is no harm in keeping it disabled in this configuration

Reviewer 3: Minor correction:
`after initial boot`

>    * It is also not necessary to disable the CloudBase-Init service after inital boot, however there is no harm in keeping it disabled in this configuration

Author: ```suggestion
   * It is also not necessary to disable the CloudBase-Init service after initial boot, however there is no harm in keeping it disabled in this configuration
```
